civihr_employee_portal/views/templates/views-view-table--Documents--block.tpl.php

```diff
@@ -78,43 +104,54 @@
           <?php endif; ?>
         <tbody>
           <?php foreach ($rows as $row_count => $row):
-              if (!isset($row['id'])) {
-                printf('<tr class = "document-row no-results"><td colspan="4">%s</td></tr>', $row[0]);
-                continue;
-              }
-              $class = 'document-row status-id-' . strtolower(str_replace(' ', '-', $row['status_id'])); ?>
+            $rowID = strip_tags(CRM_Utils_Array::value('id', $row));
+            if (!$rowID) {
+              printf('<tr class = "document-row no-results"><td colspan="4">%s</td></tr>', $row[0]);
+              continue;
+            }
+            $class = 'document-row status-id-' . strtolower(str_replace(' ', '-', $row['status_id'])); ?>
             <tr <?php if ($row_classes[$row_count] || $class) {
                 print 'class="' . implode(' ', $row_classes[$row_count]) . ' ' . $class . '"';
               } ?>>
-                <?php foreach ($row as $field => $content): ?>
-                <td <?php if ($field_classes[$field][$row_count]) {
-                print 'class="' . $field_classes[$field][$row_count] . '" ';
-              } ?><?php print drupal_attributes($field_attributes[$field][$row_count]); ?>>
                 <?php
-                if ($field === 'document_contacts' || $field === 'document_contacts_1'):
-                  print strip_tags(html_entity_decode($content));
-                  continue;
-                endif;
-                ?>
+                foreach ($row as $field => $content):
+                  $class = '';
+                  $attribute = '';
+                  if (!empty($field_classes[$field][$row_count])) {
+                    $class = sprintf('class = "%s"', $field_classes[$field][$row_count]);
+                  }
+                  if (!empty($field_attributes[$field][$row_count])) {
+                    $attribute = drupal_attributes($field_attributes[$field][$row_count]);
+                  }
 
-                <?php print $content; ?>
-                </td>
-                <?php endforeach; ?>
-              <td ct-spinner ct-spinner-id="document-<?php print strip_tags($row['id']); ?>">
-                <?php if (strip_tags($row['status_id']) == 3): ?>
+                  printf('<td %s %s>%s</td>', $class, $attribute, $content);
+                endforeach; ?>
```

Comment:
Since this is PHP only, there's no need to use the alternative syntax for the foreach loop. Also, it looks a bit weird the fact that the foreach uses the alternative syntax and the ifs don't